source/beamline/AMBeamline.h

```diff
@@ -67,22 +67,21 @@ class AMBeamline : public AMControl, public AMBeamlineControlAPI, public AMBeaml
 	static AMBeamline* bl();
 	/// Call this to delete the beamline object instance
 	static void releaseBl();
-
+	/// Destructor.
 	virtual ~AMBeamline();
 
+	/// Returns a string with a human readable text of what is important about this detector.
```

Comment:
Ah, I see. The comment confused me, but there is one function in AMDetector and another one in AMBeamline.